### ISSUE TITLE
Cassandra doesn't log to its log file

### ISSUE THREAD

**Issue by FlorianHockmann** [JanusGraph/janusgraph#291]
When I start JanusGraph with the provided `janusgraph.sh` script without modifying anything I get the following warning and Cassandra doesn't log to the default log file `log/cassandra.log`:

```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/janusgraph/janusgraph-0.1.1-hadoop2/lib/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/janusgraph/janusgraph-0.1.1-hadoop2/lib/logback-classic-1.1.2.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
```

This seems to only affect Cassandra as Gremlin Server and Elasticsearch both log to their own log files.
I manually deleted the `logback-classic-1.1.2.jar` from the `lib` directory which fixed the issue for me. So it seems that Cassandra uses by default the logback logger which is a bit odd because `conf/cassandra` contains a `log4j-cassandra.properties` file but no `logback.xml` file.
[The `pom.xml` of `janusgraph-cassandra`](https://github.com/JanusGraph/janusgraph/blob/e1b087566bbc85ecfadaef9455c4f8358ae88947/janusgraph-cassandra/pom.xml) contains a comment about this issue, but it sounds like this should only affect testing.

Is this a known problem with JanusGraph and Cassandra? And isn't there a way this can be resolved in the pre-packaged zip archive? Maybe by adding a `logback.xml` file to the `conf/cassandra` directory?

**Comment by sjudeng** [comment]
Good catch. It looks like there was a hard coded version in cassandra.in.sh. Should be fixed by #292.

**Comment by FlorianHockmann** [comment]
Thank you very much @sjudeng! That was actually much easier to fix than I thought 😄
